scripts/docs-upload-gh.sh

```diff
@@ -1,36 +0,0 @@
-#!/bin/bash
```

Comment:
are we not uploading to github pages anymore?

Reply:
we are, i just moved the logic to `scripts/upload-docs-build-gh.sh` to be consistent with the docspot upload script